src/CTA.Rules.Config/Utils.cs

```diff
@@ -15,6 +16,13 @@ namespace CTA.Rules.Config
     public class Utils
     {
         private const string DefaultMutexName = "DefaultMutex";
+        private static List<SolutionProjectType> AcceptedProjectTypes = new List<SolutionProjectType>()
```

Comment:
Please pay special attention to this list of allowed project types, the entire list is:
SolutionProjectType.SharedProject,
SolutionProjectType.SolutionFolder,
SolutionProjectType.Unknown,
The first two of which are non-buildable. Unknown is most likely not buildable.

Reply:
Why do we need a list of these types, and what happens if a project isn't of that type? Do we ignore while copying/porting?

Reply:
So without this list of types, many non-project like references get picked up within the solution file as "projects", one of the worse perpetrators of this is a folder. If there is a folder in the root solution file it gets defined as a "project" in that solution file. Which means we will attempt to build it and port it and this causes unexpected errors. Before these would just be ignored as we were scanning folder structure for .csproj type files only. So with this new code we ignore any non build-able type of project essentially. I am not sure about porting as it is still part of the solution, it just doesn't get defined as a project specifically, so I would still expect it to get copied but not to be treated as a project to run rules against.